proto/duality/dex/tick_liquidity.proto

```diff
@@ -2,24 +2,15 @@ syntax = "proto3";
 package dualitylabs.duality.dex;
 
 option go_package = "github.com/duality-labs/duality/x/dex/types";
-import "gogoproto/gogo.proto";
 import "duality/dex/limit_order_tranche.proto";
-import "duality/dex/pair_id.proto";
-
+import "duality/dex/pool_reserves.proto";
 
 message TickLiquidity {
```

Comment:
What about pulling out tickIndex, tokenIn, pairId into this area?

Reply:
Again see above comment. But also its much easier to have all data stored on the oneof type since we will be using that anyways. Storing fields on the top level requires moving them back and forth between sub-type and parent type for accessing and saving